components/03-modules/footer/_footer.scss

```diff
@@ -69,6 +72,13 @@ $footer__bottom-bar-font-size        : $font-size-small !default;
                 border-bottom: 0;
             }
         }
+
+        .list__item {
+            padding-left: $footer__link-inner-padding;
```

Comment:
Is it ok to assign variaiable `padding` to `padding-left` property?

Reply:
I think it's ok, b/c if you can use the same value and it's also "padding" why not... but I've changes some styles here, please recheck if now is ok